fcrepo-http-api/pom.xml

```diff
@@ -77,11 +77,6 @@
       <version>${project.version}</version>
     </dependency>
 
-    <dependency>
-      <groupId>org.fcrepo</groupId>
-      <artifactId>fcrepo-connector-file</artifactId>
-      <version>${project.version}</version>
```

Comment:
I can do that, but I want to ask now-- why do we need this for a test? This seems like a bad and confusing dependency.